src/runtime/hexagon/sim/driver/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+project(SIM_DEV C CXX)
+cmake_minimum_required(VERSION 3.0.2)
+
+set(CMAKE_SYSTEM_NAME "Linux")
+
+if(EXISTS ${CMAKE_CURRENT_BINARY_DIR}/config.cmake)
+  include(${CMAKE_CURRENT_BINARY_DIR}/config.cmake)
+endif()
+
+set(EXTRA_CXX_FLAGS
+  "-O2"
+  "-Wno-format"
+  "-mhvx -mhvx-length=128b"
+  "-mv66"
```

Comment:
Do you think it is better to provide one option for users but set the default value be `-mhvx-length=128b` and `-mv66`? For example, some users maybe want to use `64` bits HVX length and use `-mv60`

Reply:
This makes no difference for users.  The driver has some HVX code in it, and that's the reason the `-mhvx` option is there.  The `-mhvx-length=128b` is added because `-mhvx` requires `-mhvx-length` to be specified.

This program will be executed by the simulator, and it's the configuration of the simulator that matters.  There the actual CPU and HVX length is set to whatever the user sets in `tvm.target.hexagon(...)`.

I changed the `-mv66` to `-mv60` though, so that `sim_dev` only has v60 instructions in it.

Reply:
If `tvm.target.hexagon()` use `-mv66` but the simulator we build is `-mv60`, will we meet illegal instruction?

Reply:
`tvm.target.hexagon` sets the simulated architecture.  If it's set to `v66` the simulated CPU will be `v66', so it can run any code built for `v66` or earlier.  The driver is just a program that will be run by the simulator.